src/components/Footer/InboxForm/InboxForm.utils.ts

```diff
@@ -0,0 +1,8 @@
+import { emailRegex } from "./InboxForm.const";
+export const checkEmail = (email: string) => {
```

Comment:
to można zapisać ciut krócej:

`export const checkEmail = (email: string) => (email.match(emailRegex))`